.circleci/config.yml

```diff
@@ -22,6 +22,9 @@ jobs:
       - run:
           name: run lint
           command: yarn lint
+      - run:
```

Comment:
I was assuming that test will do prebuild of the packages. Good catch

Reply:
I removed the pretest script, it didn't seem to be working